thirdparty/download-thirdparty.sh

```diff
@@ -339,6 +339,7 @@ if [ ! -f $PATCHED_MARK ] && [ $GPERFTOOLS_SOURCE = "gperftools-gperftools-2.7"
     touch $PATCHED_MARK
 fi
 cd -
+echo "Finished patching $GPERFTOOLS_SOURCE"
 
 # patch openssl-1.0.2k
```

Comment:
The patch of openssl-1.0.x could be removed.

Reply:
Yes, this can be removed.

Reply:
File `thirdparty/patches/openssl-1.0.2k.patch` could be also removed.